ccd-definition/CaseEvent/User/UserEventsGAspec.json

```diff
@@ -5,7 +5,7 @@
     "Name": "Make an application",
     "Description": "Application created, post actions triggered",
     "DisplayOrder": 17,
-    "PreConditionState(s)": "PENDING_CASE_ISSUED;CASE_ISSUED;AWAITING_CASE_DETAILS_NOTIFICATION;AWAITING_RESPONDENT_ACKNOWLEDGEMENT;AWAITING_APPLICANT_INTENTION",
+    "PreConditionState(s)": "PENDING_CASE_ISSUED;CASE_ISSUED;AWAITING_CASE_DETAILS_NOTIFICATION;AWAITING_RESPONDENT_ACKNOWLEDGEMENT;AWAITING_APPLICANT_INTENTION;IN_MEDIATION;JUDICIAL_REFERRAL;CASE_PROGRESSION;HEARING_READINESS;PREPARE_FOR_HEARING_CONDUCT_HEARING;DECISION_OUTCOME",
```

Comment:
have we tested the creation of GA when main state has Juidicial referral? I believe only judge can make the GA when main claim has this state.